skfuzzy/control/fuzzyvariable.py

```diff
@@ -11,6 +14,41 @@
     from .ordereddict import OrderedDict
 
 
+class FuzzyVariableAdjective(object):
```

Comment:
In that case, let's go straight for the FCL standard and use "Term". In the long run that's probably the most intuitive.